docs/admin/src/widgets/variationPreviewTemplate.js

```diff
@@ -5,6 +5,10 @@ import { Tabs } from 'govuk-frontend';
 import marked from 'marked';
 import template from '../../../_includes/variation-content.html';
 
+// react-liquid (https://github.com/aquibm/react-liquid/) isn't able to `include` other files so we
+// strip out any instances of {% include icons/XXXXX.svg %}
+const cleanTemplate = template.replace( /{%\s+include\s+\/?icons\/([\w-]+)\.svg\s+%}/g, '' );
```

Comment:
If you're doing a find and replace, why don't we find and replace the icon file name with the icon SVG?

Reply:
Good thinking! I pointed them to the SVG files.